src/app/course/challenge/list-of-teams/list-of-teams.component.html

```diff
@@ -70,9 +73,7 @@ <h3 class="tui-text_h3 tui-col_stretch flex-grow">List of Teams</h3>
                                 <ng-template #notInTeam>
                                     <button
                                         *ngIf=
-                                            "!isInTeam(team)
-                                            && !(event.is_open || event.is_closed)
-                                            && isInvited(team)"
+                                            "!isInTeam(team) && !(event.is_open || event.is_closed)"
```

Comment:
Also I think `!(event.is_open || event.is_closed)` is equivalent to `event.is_not_available_yet`.